src/main/java/core/basesyntax/strategyimpl/ReturnHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategyimpl;
+
+import core.basesyntax.storage.Storage;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnHandler implements OperationHandler {
+    @Override
+    public void calculateValue(String fruit, int quantity) {
+        if (Storage.fruits.containsKey(fruit)) {
+            Storage.fruits.put(fruit, Storage.fruits.get(fruit) + quantity);
+            return;
+        }
+        Storage.fruits.put(fruit, quantity);
+    }
```

Comment:
again what if someone tries to return fruits that you shop never sold?
But this is more question to mentor - should we handle such situation in current task without proper DB?
Personaly I made a check but it limited to check based on operations b or s where commited but it also maybe wrong
so I think its ok here